ast/Helpers.h

```diff
@@ -409,7 +409,8 @@ class MK {
     }
 
     static ExpressionPtr T_Boolean(core::LocOffsets loc) {
-        return UnresolvedConstantParts(loc, {core::Names::Constants::T(), core::Names::Constants::Boolean()});
+        static constexpr core::NameRef parts[2] = {core::Names::Constants::T(), core::Names::Constants::Boolean()};
```

Comment:
Is it worth using `std::array` here?

Reply:
I have never been that big on `std::array`, personally, but yes, it would be possible to do that here.  I am going to leave that for a potential followup.